gammapy/time/plot.py

```diff
@@ -30,3 +28,140 @@ def plot_time_difference_distribution(time, ax=None):
 
     # TODO: implement!
     raise NotImplementedError
+
+def plot_light_curve(name_3FGL, time_start, time_end, ax = None):
```

Comment:
Rename function to the more explicit `plot_fermi_3fgl_light_curve`.

Add default values `time_start=None` and `time_end=None` and document / implement them to mean "full available time range".